ui/src/pages/clearing/ClearingOfferModal.tsx

```diff
@@ -85,7 +85,7 @@ const ClearingOfferModal: React.FC<ServicePageProps<Service> & OfferProps> = ({
     const accountRequest: RequestOpenAccount = {
       accountId: {
         signatories: makeDamlSet([service.payload.provider, service.payload.customer]),
-        label: `${party}-${clearingProvider}-clearing`,
+        label: `${getName(party)}-${getName(clearingProvider)}-clearing`,
         version: '0',
```

Comment:
This reminds me of `makeAccountName` where u check `isHubDeployment`, but you don't do it here.. curious why?

Reply:
yep, same thing... I'll reuse that function